packages/kolibri-tools/lib/i18n/fonts.py

```diff
@@ -19,17 +19,28 @@
 import noto_source
 import utils
 
+logging.basicConfig(format="%(levelname)s: %(message)s", level=logging.INFO)
+logging.getLogger("fontTools").setLevel(logging.WARNING)
+logging.StreamHandler(sys.stdout)
+
+
+font_tools_version = "4.49.0"
+
 try:
     import fontTools  # noqa
-except ImportError:
-    utils.install_requirement("fonttools==4.28.1")
 
-from fontTools import merge
-from fontTools import subset
+    assert fontTools.version == font_tools_version
+except ImportError:
+    utils.install_requirement(f"fonttools=={font_tools_version}")
+except AssertionError:
+    utils.install_requirement(f"fonttools=={font_tools_version}")
+    logging.error(
+        "Wrong fontTools version was installed. This has been updated. Please re-run the command."
+    )
+    sys.exit(1)
```

Comment:
Why do we notify that we've updated fontTools on one type of error but not the other?

Reply:
The assertion error only happens in the case that fontTools was already installed. Unfortunately, because we already imported it, we have to exit Python before we can import the updated version (although, now I think about it, it _might_ be possible to clear the module cache - but generally safer to quit). When the import didn't even succeed, we can reimport again.